appium-tests/common/common.spec.js

```diff
@@ -242,7 +248,6 @@ describe('Contacts Android', function () {
 
     describe('Picking contacts', function () {
         afterEach(function (done) {
-            checkSession(done);
```

Comment:
Sigh, yes, you are correct. I don't know why I keep butting my head against this. Thank you for your patience. Re-added once more.